### PR TITLE
Release notes for BoTorch 0.8.1

### PR DESCRIPTION
**Pull request by esantorella**
## Test Plan

Looked at it in a markdown viewer

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,45 @@
 
 The release log for BoTorch.
 
+## [0.8.1] - Jan 5, 2023
+
+### Highlights
+* This release includes changes for compatibility with the newest versions of linear_operator and gpytorch.
+* Several acquisition functions now have "Log" counterparts, which provide better
+numerical behavior for improvement-based acquisition functions in areas where the probability of
+imporvement is low. For example, `LogExpectedImprovement` (#1565) should be more accurate than
```
Comment (saitcakmak) [inline CHANGELOG.md:11]:
```suggestion
improvement is low. For example, `LogExpectedImprovement` (#1565) should be more accurate than
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,45 @@
...
+* Removal of `_fit_multioutput_independent` and `allclose_mll` (#1570).
+* Better numerical behavior for fully Bayesian models (#1576).
+* More verbose Scipy `minimize` failure messages (#1579).
+* Lower-bound noise in`SaasPyroModel` to avoid Cholesky errors (#1586).
+
+#### Bug fixes
+* Error rather than failing silently for NaN values in box decomposition (#1554).
+* Fix jitter in `_psd_safe_pyro_mvn_sample` (#1564).
+* Make `get_bounds_as_ndarray` device-safe (#1567).
+
```
Comment (saitcakmak) [inline CHANGELOG.md:43]:
```suggestion


```

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,45 @@
...
+#### Other changes
+* Docstring corrections (#1551, #1557, #1573).
+* Removal of `_fit_multioutput_independent` and `allclose_mll` (#1570).
+* Better numerical behavior for fully Bayesian models (#1576).
+* More verbose Scipy `minimize` failure messages (#1579).
+* Lower-bound noise in`SaasPyroModel` to avoid Cholesky errors (#1586).
+
+#### Bug fixes
+* Error rather than failing silently for NaN values in box decomposition (#1554).
+* Fix jitter in `_psd_safe_pyro_mvn_sample` (#1564).
```
Comment (saitcakmak) [inline CHANGELOG.md:41]: `_psd_safe_pyro_mvn_sample` got removed, so no need to include this.

**Review by saitcakmak — APPROVED** [review]
Thanks! Some comments in line.

File: CHANGELOG.md (hunk 3 above)
Comment (esantorella) [inline CHANGELOG.md:41]:
```suggestion
```

File: CHANGELOG.md (hunk 1 above)
Comment (esantorella) [reply CHANGELOG.md:11]: Yep I'm sure that's right.

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,45 @@
 
 The release log for BoTorch.
 
+## [0.8.1] - Jan 5, 2023
+
+### Highlights
+* This release includes changes for compatibility with the newest versions of linear_operator and gpytorch.
+* Several acquisition functions now have "Log" counterparts, which provide better
+numerical behavior for improvement-based acquisition functions in areas where the probability of
+improvement is low. For example, `LogExpectedImprovement` (#1565) should be more accurate than
```
Comment (esantorella) [inline CHANGELOG.md:11]:
```suggestion
improvement is low. For example, `LogExpectedImprovement` (#1565) should behave better than
```

**Pull request closed without merging**